tests/integration/standard/test_rack_aware_policy.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+
+from cassandra.cluster import Cluster, ExecutionProfile, EXEC_PROFILE_DEFAULT, NoHostAvailable
+from cassandra.policies import ConstantReconnectionPolicy, HostFilterPolicy, RackAwareRoundRobinPolicy, RoundRobinPolicy,  SimpleConvictionPolicy, \
+    WhiteListRoundRobinPolicy, ExponentialBackoffRetryPolicy
+from cassandra.pool import Host
+from cassandra.connection import DefaultEndPoint
+
+from tests.integration import PROTOCOL_VERSION, local, use_multidc, use_singledc, TestCluster
+
+from concurrent.futures import wait as wait_futures
+
+
+def setup_module():
+    use_multidc({'DC1': {'RC1': 2, 'RC2': 2}, 'DC2': {'RC1': 2}})
+
+class RackAwareRoundRobinPolicyTests(unittest.TestCase):
+    @classmethod
+    def setup_class(cls):
+        cls.cluster = Cluster(contact_points=["127.0.0.1", "127.0.0.2", "127.0.0.3", "127.0.0.4", "127.0.0.5", "127.0.0.6"], protocol_version=PROTOCOL_VERSION,
+                              load_balancing_policy=RackAwareRoundRobinPolicy("DC1", "RAC1", used_hosts_per_remote_dc=1),
+                              reconnection_policy=ConstantReconnectionPolicy(1))
+        cls.session = cls.cluster.connect()
+
+    @classmethod
+    def teardown_class(cls):
+        cls.cluster.shutdown()
+
+    def test_rack_aware(self):
+        self.cluster = Cluster(contact_points=["127.0.0.1", "127.0.0.2", "127.0.0.3", "127.0.0.4", "127.0.0.5", "127.0.0.6"], protocol_version=PROTOCOL_VERSION,
+                              load_balancing_policy=RackAwareRoundRobinPolicy("DC1", "RAC1", used_hosts_per_remote_dc=1),
+                              reconnection_policy=ConstantReconnectionPolicy(1))
+        self.session = self.cluster.connect()
+        queried_hosts = set()
+        for _ in range(10):
+            response = self.session.execute('SELECT * from system.local')
+            queried_hosts.update(response.response_future.attempted_hosts)
+        queried_hosts = set(host.address for host in queried_hosts)
+        # Performe some checks
```

Comment:
I think we can use a similar approach as in shard aware tests, and use tracing to validate requests arrived at the correct nodes